.docker/fedora/Dockerfile.sh

```diff
@@ -0,0 +1,51 @@
+cat <<EOF
+FROM docker.io/oi4ai/bout-base:openmp-$MPI-mini
```

Comment:
Rather than having this as a shell script, could we make use of `ARG`/`build_arg`? See e.g. https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

Reply:
I should have read the official documentation - and not some random stuff on the internet :-(

I thought it wouldn't be possible, but as that seems to be the case, I can go back to the old version ...